src/components/Checkbox/Checkbox.js

```diff
@@ -130,4 +163,6 @@ Checkbox.defaultProps = {
   label2: 'Checkbox 2 label',
   label3: 'Checkbox 3 label',
   required: 'none',
+  inline: false,
+  grid: false,
```

Comment:
How do the designers control number of columns in the grid?
The difference between the Grid property and Inline is that one renders **all** the checkboxes inline and grid renders them in a controlled grid with predefined number of columns